packages/esm-patient-registration-app/src/patient-registration/validation/patient-registration-validation.test.tsx

```diff
@@ -1,129 +1,143 @@
-import React from 'react';
-import { render, fireEvent, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import { Formik, Form } from 'formik';
-import { useConfig } from '@openmrs/esm-framework';
 import { validationSchema } from './patient-registration-validation';
-import { NameField } from '../field/name/name-field.component';
-import { PatientRegistrationContext } from '../patient-registration-context';
-import { initialFormValues } from '../patient-registration.component';
-import { FormValues } from '../patient-registration-types';
 
-const mockUseConfig = useConfig as jest.Mock;
-const mockFieldConfigs = {
-  fieldConfigurations: {
-    name: {
-      displayMiddleName: true,
-    },
-  },
-};
-
-jest.mock('@openmrs/esm-framework', () => {
-  const originalModule = jest.requireActual('@openmrs/esm-framework');
+describe('Patient Registration Validation', () => {
+  describe('validationSchema', () => {
+    const validFormValues = {
+      givenName: 'John',
+      familyName: 'Doe',
+      additionalGivenName: '',
+      additionalFamilyName: '',
+      gender: 'Male',
+      birthdate: new Date('1990-01-01'),
+      birthdateEstimated: false,
+      deathDate: null,
+      email: 'john.doe@example.com',
+      identifiers: {
+        nationalId: {
+          required: true,
+          identifierValue: '123456789',
+        },
+        passportId: {
+          required: false,
+          identifierValue: '',
+        },
+      },
+    };
 
-  return {
-    ...originalModule,
-    useConfig: jest.fn(),
-    validator: jest.fn(),
-  };
-});
+    const validateFormValues = async (formValues) => {
```

Comment:
We can do something like this:
```
const validateFormValues = async (formValues) => {
  try {
    await validationSchema.validate(formValues, { abortEarly: false });
  } catch (err) {
    return err;
  }
};
```

Reply:
@Piumal1999 updated the PR please review